### DISCUSSION TITLE
Autocompletion like warp.dev

### DISCUSSION THREAD

**Discussion by marcelobns** [community/community#16176] [Copilot Conversations]
<img width="212" alt="Screen Shot 2022-05-06 at 21 39 35" src="https://user-images.githubusercontent.com/1869500/167232928-1ac4a1db-f8d2-4e06-895d-3465d0d81e8b.png">

You could complete with the hole sentence or just some words or even letters.

Less confuse and more productive. 

try it..
https://www.warp.dev/

**Comment by bamar951** [comment]
[admob-report.csv](https://github.com/github/feedback/files/8654123/admob-report.csv)
